prep_data/tabular_data/01_preprocessing_tabular_data.ipynb

```diff
@@ -6,14 +6,13 @@
    "source": [
```

Comment:
<p>plotly is not pre-installed. see build-error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2002/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Plotly gets pip installed in the first notebook but not in the other two notebooks. Also, CI cannot coordinate the build sequentially across the notebooks one after the other, so this will break CI.